airbyte-integrations/connectors/source-getlago/manifest.yaml

```diff
@@ -318,6 +318,17 @@ definitions:
           $ref: "#/definitions/base_requester"
           path: /customers/{{stream_slice.customer_external_id}}/current_usage
           http_method: GET
+          error_handler:
```

Comment:
@Sviande, can you update the connector version in the metadata.yaml and pyproject.toml files? Additionally, please add an entry in the changelog connector docs (docs/integrations/source/...).